HW4/Anastasiia-web/1.py

```diff
@@ -0,0 +1,62 @@
+# Factorial
```

Comment:
num=int(input("Input natural number: "))

fact = 1

if num == 1 or num == 0:
    print("Factorial {}! =".format(num), fact)
elif num<0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,num+1):
        fact *= item
    print("Factorial {}! =".format(num), fact)